### PR TITLE
typo: Provder -> Provider

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: docs/framework/data/adonet/ef/provider-manifest-specification.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ This section discusses how a data store provider can support the types and funct
  The provider manifest is loaded by the Store Metadata loader (StoreItemCollection), either by using a data store connection or a provider manifest token.  
   
 #### Using a Data Store Connection  
- When the data store connection is available, call DbProvderServices.GetProviderManifestToken to return the token that is passed to the GetProviderManifest method, which returns DbProviderManifest. This method delegates to the provider's implementation of GetDbProviderManifestToken.  
+ When the data store connection is available, call DbProviderServices.GetProviderManifestToken to return the token that is passed to the GetProviderManifest method, which returns DbProviderManifest. This method delegates to the provider's implementation of GetDbProviderManifestToken.  
   
 ```
```
Comment (mairaw) [inline docs/framework/data/adonet/ef/provider-manifest-specification.md]:
```suggestion
```csharp
```

File: docs/framework/data/adonet/ef/provider-manifest-specification.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ This section discusses how a data store provider can support the types and funct
  The provider manifest is loaded by the Store Metadata loader (StoreItemCollection), either by using a data store connection or a provider manifest token.  
   
 #### Using a Data Store Connection  
- When the data store connection is available, call DbProvderServices.GetProviderManifestToken to return the token that is passed to the GetProviderManifest method, which returns DbProviderManifest. This method delegates to the provider's implementation of GetDbProviderManifestToken.  
+ When the data store connection is available, call DbProviderServices.GetProviderManifestToken to return the token that is passed to the GetProviderManifest method, which returns DbProviderManifest. This method delegates to the provider's implementation of GetDbProviderManifestToken.
```
Comment (mairaw) [inline docs/framework/data/adonet/ef/provider-manifest-specification.md]:
```suggestion
 When the data store connection is available, call <xref:System.Data.Common.DbProviderServices.GetProviderManifestToken?displayProperty=nameWithType> to return the token that is passed to the <xref:System.Data.Common.GetProviderManifest%2A> method, which returns <xref:System.Data.Common.DbProviderManifest>. This method delegates to the provider's implementation of `GetDbProviderManifestToken`.  
```

**Review by mairaw — APPROVED** [review]
Looks good. Left some suggestions to improve the formatting a bit.

**Review by mairaw — CHANGES_REQUESTED** [review]
Selected the wrong option before.

File: docs/framework/data/adonet/ef/provider-manifest-specification.md [hunk 3]
```diff
@@ -77,9 +77,9 @@ This section discusses how a data store provider can support the types and funct
  The provider manifest is loaded by the Store Metadata loader (StoreItemCollection), either by using a data store connection or a provider manifest token.  
   
 #### Using a Data Store Connection  
- When the data store connection is available, call DbProvderServices.GetProviderManifestToken to return the token that is passed to the GetProviderManifest method, which returns DbProviderManifest. This method delegates to the provider's implementation of GetDbProviderManifestToken.  
+ When the data store connection is available, call <xref:System.Data.Common.DbProviderServices.GetProviderManifestToken?displayProperty=nameWithType> to return the token that is passed to the <xref:System.Data.Common.GetProviderManifest%2A> method, which returns <xref:System.Data.Common.DbProviderManifest>. This method delegates to the provider's implementation of `GetDbProviderManifestToken`.
```
Comment (nschonni) [inline docs/framework/data/adonet/ef/provider-manifest-specification.md]: I's complaining about the new xrefs. Maybe the `%2A` at the end of the second one?

Comment (rpetrusha) [inline docs/framework/data/adonet/ef/provider-manifest-specification.md]:
```suggestion
 When the data store connection is available, call <xref:System.Data.Common.DbProviderServices.GetProviderManifestToken%2A?displayProperty=nameWithType> to return the token that is passed to the <xref:System.Data.Common.DbProviderServices.GetProviderManifest%2A> method, which returns <xref:System.Data.Common.DbProviderManifest>. This method delegates to the provider's implementation of `GetDbProviderManifestToken`.  
```

**Review by rpetrusha — APPROVED** [review]
I think that my suggestions fix the 2 broken xrefs, @nschonni.

File: docs/framework/data/adonet/ef/provider-manifest-specification.md (hunk 3 above)
Comment (nschonni) [reply docs/framework/data/adonet/ef/provider-manifest-specification.md]: Looks like that worked

**Comment by rpetrusha** [comment]
Yes, it does fix the broken xrefs, @nschonni. Do you want to approve the PR now, @mairaw?

**Review by mairaw — APPROVED** [review]
Yes, thanks for your help as well @rpetrusha and apologies for giving the wrong links in the first place.

**Pull request merged**